molr-mole-core/src/main/java/io/molr/mole/core/support/OngoingMissionRun.java

```diff
@@ -1,32 +1,37 @@
 package io.molr.mole.core.support;
 
-import io.molr.commons.domain.MissionHandle;
-import io.molr.commons.domain.MissionState;
-import io.molr.commons.domain.Result;
-import io.molr.commons.domain.RunState;
+
+import io.molr.commons.domain.*;
 import io.molr.mole.core.api.Mole;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 import java.time.Duration;
+import java.util.function.Function;
 
 import static java.util.Objects.requireNonNull;
 
+/**
+ *
+ */
 public class OngoingMissionRun {
 
-    private final Mole mole;
-    private final Mono<MissionHandle> handle;
+    protected final Mole mole;
+    protected final Mono<MissionHandle> handle;
 
     public OngoingMissionRun(Mole mole, Mono<MissionHandle> handle) {
-        this.mole = requireNonNull(mole, "agency must not be null");
+        this.mole = requireNonNull(mole, "mole must not be null");
         this.handle = requireNonNull(handle, "handle must not be null");
     }
 
+    /**
+     * @return {@link OngoingMissionRun}
+     */
     public OngoingMissionRun and() {
         return this;
     }
 
-    public Mono<MissionHandle> forget() {
-        return this.handle;
+    public void forget() {
```

Comment:
Consider documenting why this method is there (because a useful 'clause' with method chaining is allowes...)